demo/backend/_includes/templates/root.xml.njk

```diff
@@ -0,0 +1,3 @@
+{# Base template that should be used for all screens. Shows the Header + back button #}
```

Comment:
comment is outdated - perhaps its worth using this template as a root for the `doc` element and reuse in `base.xml` too?
```suggestion
{# Base template that should be used for all documents #}
```

Reply:
Good catch, thanks. Yes, I was thinking I may as well make it root for everything.

https://github.com/Instawork/hyperview/pull/1005/commits/98355ce2a6d098de553ba4f75926ba5506c4d011

https://github.com/Instawork/hyperview/pull/1005/commits/6ccb5887a4a2255def3a35798a1adfa60136d393

Reply:
are your latest edit meant to address this or not? The root file is empty right now, my previous comment suggested to have:
```xml
<doc xmlns="https://hyperview.org/hyperview" {% block custom_ns %}{% endblock %}>
  {% block root %}{% endblock %}
</doc>
```

then de-dup the `doc` element from `demo/backend/_includes/templates/base.xml.njk` and `demo/backend/index.xml.njk`. Minor but since you're touching this code we may as well get that done.